README.md

```diff
@@ -0,0 +1,75 @@
+
```

Comment:
I guess I'm cool with the readme being HTML but markdown is so much simpler 😂

Reply:
Most of it is markdown still, just some fancy dropdown menu at the top with a spot for a logo